modules/ROOT/pages/try-scope-concept.adoc

```diff
@@ -49,6 +49,90 @@ A new transaction is started every time the scope is executed.
 +
 Only relevant in cases where execution order may vary (for example, due to asynchronous actions occurring outside the flow): If the current processing of the flow has already begun a transaction, join it. Otherwise, begin a new one.
 
+=== Analyzing different Transactional Actions in Try Scope
+
+Supponse we have the next configuration: 
+
+[source,xml,linenums]
+----
+<flow name="someFlow">
+	<jms:listener config-ref="JMS_Config" destination="test.in" transactionalAction="ALWAYS_BEGIN"/>
+	<!-- Processors -->
+	<try transactionalAction="?">
+		<!-- Processors -->
+		<!-- Join if possible is the default value for jms:publish operation --> 
+		<jms:publish config-ref="JMS_Config" destination="test.out"/>
+		<raise-error type="APP:SOME"/>
+		<!-- More Processors -->
+	</try>
+	<!-- Processors -->
+</flow>
+----
+
+As expected, in case there are no processors that produces any error this will finish the execution and commit the transaction, independently of the `transactionalAction` set in the Try Scope. 
```

Comment:
If the processors within the Try Scope do not produce any error, the scope finishes the execution and commits the transaction, independently of the configured `transactionalAction` value.